src/components/blockquote/blockquote.scss

```diff
@@ -62,23 +70,171 @@ blockquote {
       position: absolute;
       content: '';
       height: 10px;
-      top: 15px;
+      top: 11px;
       margin: 0 auto;
       left: 0;
       right: 0;
-      width: 15%;
+      width: 130px;
       background: $primary;
       -webkit-transition: 0.5s;
       transition: 0.5s;
     }
   }
-
   // @todo 2020/12/08 Remove '.blockquote-right' class in v4.0.0
-  .blockquote-right &,
-  &.blockquote--right {
-    text-align: right;
+  &.blockquote--right,
+  .blockquote-right & {
+    text-align: end;
     border-left: none;
-    border-right: 8px solid $primary;
+    border-right: 10px solid $primary;
     @include padding($top: $xsm, $bottom: $xsm, $left: 0, $right: $lg);
   }
 }
+
+// Shared styles for above and below options.
+blockquote.blockquote--img-above,
+blockquote.blockquote--img-below {
```

Comment:
Is there a reason not to include this inside the previous `blockquote` section?

Reply:
Good question/observation, I'll see if I can refactor that!

Reply:
This should be resolved if you have time to take another look.